packages/frontend/src/app/services/utils.service.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Injectable} from '@angular/core';
+import {DomSanitizer, SafeUrl} from '@angular/platform-browser';
+import {environment} from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UtilsService {
+  constructor(public sanitizer: DomSanitizer) {}
+
+  parseUrl(url: string): SafeUrl {
+    // TODO: should be done on the server ideally
+    url = url?.replace('git://', 'https://');
```

Comment:
I think the logic for turning a `git://` address into its repository address is a bit more nuanced:

https://www.npmjs.com/package/github-url-from-git

Reply:
@StuartRucker do you think we can do this in the api ? That way the client receives links only.